### ISSUE TITLE
HmIP-SCTH230 - Formatting of ppm value

### ISSUE THREAD

**Issue by eikowagenknecht** [SukramJ/hahomematic#158]
**custom_component/hahomematic version (if applicable):**
latest

**Home Assistant version (if applicable):**
latest

**CCU version:**
CCU3 RaspberryMatic

**Describe the bug**
![image](https://user-images.githubusercontent.com/1475672/144467631-343a4df4-9c99-47fb-929f-0b89ab2a0dd4.png)
The upper two sensors are from the *Netatmo* integration, the third the HmIP-SCTH230 from *hahomematic*.

As you can see, there is always an appended ",0" for the value, suggesting more precision than it has.

**Expected behavior**
In the CCU only integers are shown:
![image](https://user-images.githubusercontent.com/1475672/144468262-84540def-eb83-4d45-9288-683c3b55a2cb.png)

As this is fake precision, it would be better to use integers as well.

**Additional context**
See https://github.com/danielperna84/pyhomematic/issues/449 for the original discussion, especially the last two comments:

> So it's technically correct to add the trailing zero, I see. But doesn't make much sense because the device only ever returns values with a leading zero and even the original web interface doesn't show it (see screenshot above). Any idea how this could be handled? Is there a possibility included in hahomematic for device specific workarounds for this kind of weirdness?

Response from danielperna84 
> ...Feel free to create an issue for hahomematic. Maybe Markus has an idea on how to solve this elegantly... We can't use any generic approach for this like always converting floats to ints, because obviously other float-based parameters make use of the decimals. So this would be some sort of custom device (like blinds or thermostats).

**Comment by SukramJ** [comment]
This is a FLOAT based sensor, so the displayed value is what is stored in the CCU. We don't modify values and append a trailing zero, and i dont't know why the CCU displays a rounded value.
@danielperna84 I don't think we should have something like custom sensors just for rounding issues.

Maybe this is worth a feature request in HA: Round Value / convert type based on unit or device_class.

This is currently configured for the CONCENTRATION param (like in the old integration):
```
        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
        device_class=SensorDeviceClass.CO2,
```

**Comment by danielperna84** [comment]
> @danielperna84 I don't think we should have something like custom sensors just for rounding issues.

I agree. I just didn't want to decline the request straight away without getting your opinion. After all there might be other cases where a transparent conversion might make sense. Like f.ex. the energy-sensors which are exposed as `Wh` from the devices, but the typical unit would be `kWh`. Hence the idea of at least a general framework which would allow doing such things in case it is needed.

**Comment by SukramJ** [comment]
Fixed by 0.23.0

**Comment by eikowagenknecht** [comment]
Wow, didn't think this would make it in. Thank you!
